debian/Dockerfile

```diff
@@ -4,6 +4,11 @@ FROM debian:testing
 
 MAINTAINER "R-hub admin" admin@r-hub.io
 
+
+## 2 lines copied from https://github.com/rocker-org/rocker/blob/749e7e9ce62a242113f447f93904e9faff7df05e/r-devel/Dockerfile#L94
```

Comment:
Can you please remove these comments? Thanks.

Reply:
:heavy_check_mark: